lib/discover/client_test.go

```diff
@@ -1,239 +0,0 @@
-// Copyright (C) 2014 The Syncthing Authors.
```

Comment:
I hope to add one or two back again. Otherwise I'll feel bad for the kittens. :(